libs/agno/agno/eval/accuracy.py

```diff
@@ -22,6 +27,20 @@ class AccuracyAgentResponse(BaseModel):
     accuracy_reason: str = Field(..., description="Detailed reasoning for the accuracy score.")
 
 
+class AccuracyEvalMode(str, Enum):
+    """
+    The complexity mode used for the evaluation.
+
+    BASIC: The evaluation is performed using a simple comparison of the expected output and the Agent's output.
+    AGENTIC: The evaluation is performed by the evaluator agent, comparing the Agent's output and the expected output.
+    REASONING: The evaluation is performed by the evaluator agent, analyzing the Agent's output rationale, completeness and correctness, apart from accuracy.
+    """
+
+    BASIC = "basic"
+    AGENTIC = "agentic"
+    REASONING = "reasoning"
```

Comment:
I'm wondering whether we should do `basic` and `agentic` only, and you can specify `reasoning=True` to do agentic+reasoning?